.pre-commit-config.yaml

```diff
@@ -31,7 +31,7 @@ repos:
     args:
     - --builtin-schema
     - github-workflows-require-timeout
-    files: ^\.github/workflows/[^/]+$
+    files: ^\.github\/workflows/[^/]+$
```

Comment:
Could you submit this separately?

Reply:
I've removed this change. The existing regex (`^\.github/workflows/[^/]+$`) seems like it should work fine, but pre-commit was matching files in `.github/actions/` and raising errors. Dynamically generating the entire `action.yml` as you suggested (https://github.com/pypa/gh-action-pypi-publish/pull/230/files#r1603950013) will avoid the problem.